testing/correctness/tests/cli_tests/Makefile

```diff
@@ -0,0 +1,52 @@
+# include root makefile
+ifndef ROOT_MAKEFILE_MK
+include ../../../../Makefile
+endif
+
+# prevent rules from being evaluated/included multiple times
+ifndef $(abspath $(lastword $(MAKEFILE_LIST)))_MK
+$(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
+
+# The following are control variables that determine what logic from `rules.mk` is enabled
+
+# `true`/`false` to enable/disable the actual unit test command so it can be overridden (the targets are still created)
+# applies to both the pony and elixir test targets
+$(abspath $(lastword $(MAKEFILE_LIST)))_UNIT_TEST_COMMAND := false
+
+# `true`/`false` to enable/disable generate pony related targets (build/test/clean) for pony sources in this directory
+# otherwise targets only get created if there are pony sources (*.pony) in this directory.
+$(abspath $(lastword $(MAKEFILE_LIST)))_PONY_TARGET := false
+
+# `true`/`false` to enable/disable generate final file build target using ponyc command for the pony build target so
+# it can be overridden manually
+$(abspath $(lastword $(MAKEFILE_LIST)))_PONYC_TARGET := false
+
+# `true`/`false` to enable/disable generate exs related targets (build/test/clean) for elixir sources in this directory
+# otherwise targets only get created if there are elixir sources (*.exs) in this directory.
+$(abspath $(lastword $(MAKEFILE_LIST)))_EXS_TARGET := false
+
+# `true`/`false` to enable/disable generate docker related targets (build/push) for a Dockerfile in this directory
+# otherwise targets only get created if there is a Dockerfile in this directory
+$(abspath $(lastword $(MAKEFILE_LIST)))_DOCKER_TARGET := false
+
+# `true`/`false` to enable/disable recursing into Makefiles of subdirectories if they exist
+# (and by recursion every makefile in the tree that is referenced)
+$(abspath $(lastword $(MAKEFILE_LIST)))_RECURSE_SUBMAKEFILES := false
+
+CLI_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+cli_tests: CUSTOM_PYTHONPATH = $(CLI_PATH):$(DUMMY_PYTHON_PATH)
+
+# standard rules generation makefile
+include $(rules_mk_path)
+
+build-testing-correctness-tests-cli_tests: build-machida
+build-testing-correctness-tests-cli_tests: build-testing-tools-external_sender
+integration-tests-testing-correctness-tests-cli_tests: cli_tests
+
+cli_tests:
+	cd $(CLI_PATH) && \
+	python2 -m pytest -s --color=yes --tb=native \
+	 --verbose --exitfirst cli_tests.py \
+	$(pytest_exp)
```

Comment:
You left a `-s`.
There's a settings ini file in `$(integration_path)` that can make this command a bit cleaner:
```
python2 -m pytest -c $(integration_path)/pytest.ini cli_tests.py $(pytest_exp)
```